hassio/src/dialogs/backup/dialog-hassio-backup-upload.ts

```diff
@@ -64,6 +64,7 @@ export class DialogHassioBackupUpload
               .path=${mdiClose}
               slot="actionItems"
               dialogAction="cancel"
+              dialogInitialFocus
```

Comment:
The primary element here is hassio-upload-backup, why is that not the one we focus on?

Reply:
It's irrelevant mostly.  This dialog opens another to pick a file which immediately gets focus.  If you pick one, this is closed and a backup details dialog is opened.  If you hit cancel, then you have to close this dialog too.